template/en/default/pages/release-notes3.html.tmpl

```diff
@@ -3312,11 +3312,11 @@
 <p>This gives [% terms.Bugzilla %] very advanced plugin support. You can
   hook templates, hook code, add new parameters, and use the XML-RPC
   interface. So we'd like to see some [% terms.Bugzilla %] plugins
-  written! Let us know on the <a href="http://bugzilla.org/cgi-bin/mj_wwwusr?func=lists-long-full&amp;extra=developers">developers&#64;bugzilla.org</a>
+  written! Let us know on the <a href="https://lists.bugzilla.org/cgi-bin/mj_wwwusr?func=lists-long-full&amp;extra=developers">developers&#64;bugzilla.org</a>
   mailing list if you write a plugin.</p>
 
 <p>If you need more hooks, please 
-  <a href="http://www.bugzilla.org/developers/reporting_bugs.html">File a b<!-- -->ug</a>!</p>
+  <a href="https://www.bugzilla.org/developers/reporting_bugs.html">File a b<!-- -->ug</a>!</p>
```

Comment:
`b<!-- -->ug` should be replaced by [% terms.bug %], right?

Reply:
Wasn't aware of that, but it's out of scope I think.

Reply:
Since it is linking to an extern system, using the comment hack is fine.